code/__defines/items_clothing.dm

```diff
@@ -128,17 +128,17 @@
 #define LOWER_TORSO 0x10
 #define LEG_LEFT    0x20
 #define LEG_RIGHT   0x40
-#define LEGS        0x60   //  LEG_LEFT | LEG_RIGHT
 #define FOOT_LEFT   0x80
 #define FOOT_RIGHT  0x100
-#define FEET        0x180  // FOOT_LEFT | FOOT_RIGHT
 #define ARM_LEFT    0x200
 #define ARM_RIGHT   0x400
-#define ARMS        0x600 //  ARM_LEFT | ARM_RIGHT
 #define HAND_LEFT   0x800
 #define HAND_RIGHT  0x1000
-#define HANDS       0x1800 // HAND_LEFT | HAND_RIGHT
-#define FULL_BODY   0xFFFF
+#define LEGS        0x60	//  LEG_LEFT | LEG_RIGHT
+#define FEET        0x180	// FOOT_LEFT | FOOT_RIGHT
+#define ARMS        0x600	//  ARM_LEFT | ARM_RIGHT
+#define HANDS       0x1800	// HAND_LEFT | HAND_RIGHT
+#define FULL_BODY   0x1FFF	// EVERYTHING
```

Comment:
Why did you change this?

Reply:
Leftover change from something else. It used to be 0xFFFF but that actually didn't mean FULL_BODY like the other combinations had